.github/workflows/sycl_linux_build_and_test.yml

```
@@ -169,13 +169,19 @@ jobs:
         cmake --build $GITHUB_WORKSPACE/build --target utils/not/install
         cmake --build $GITHUB_WORKSPACE/build --target utils/lit/install
         cmake --build $GITHUB_WORKSPACE/build --target utils/llvm-lit/install
-        cmake --build $GITHUB_WORKSPACE/build --target install-clang-format
-        cmake --build $GITHUB_WORKSPACE/build --target install-clang-tidy
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-size
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-cov
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-profdata
         cmake --build $GITHUB_WORKSPACE/build --target install-compiler-rt
 
+    - name: Install clang-format/clang-tidy
+      # We install these into our nightly container that is later used in CI to
+      # run lint task.
```

```suggestion
      # We install these into our nightly container that CI uses to run lint
      #  checks.
```
I heard that passive voice is difficult to read. :)